hugegraph-llm/src/hugegraph_llm/utils/vector_index_utils.py

```diff
@@ -70,6 +75,8 @@ def clean_vector_index():
 
 
 def build_vector_index(input_file, input_text):
+    if input_file and input_text:
+        raise gr.Error("Please only choose one between file and text.")
     texts = read_documents(input_file, input_text)
     builder = KgBuilder(LLMs().get_chat_llm(), Embeddings().get_embedding(), get_hg_client())
```

Comment:
Maybe we should transfer the default file demo to `text`?

Avoid always needing to manually delete it

Reply:
OK!